lib/PolynomialTermOperations.js

```diff
@@ -107,24 +111,150 @@ function addlikeTermPolynomialNodes(node) {
   if (!canAddLikeTermPolynomialNodes(node)) {
     return NodeStatus.noChange(node);
   }
-  const polynomialTermList = node.args.map(n => new PolynomialTermNode(n));
 
-  // get a list of the coefficients, to be added
+  let oldNode = clone(node, false);
+  const subSteps = [];
+
+  // STEP 1: IF any nodes have no coefficient, make it have coefficient 1
```

Comment:
to try to make it clear that that step doesn't always happen, but I don't think it's very effective :p I'll update it and step 2 as well